appd.adoc

```diff
@@ -20,89 +19,63 @@ The variables containing the terms may optionally have `standard_name` attribute
 The `standard_name` of the dimensional coordinate values which are computed by the formula may optionally be specified by the `computed_standard_name` attribute of the vertical coordinate variable, as indicated in this Appendix.
 A `computed_standard_name` is uniquely implied by the formula in some cases, while in others it depends on the `standard_name` of one or more of the terms, with which it must be consistent.
 
-
 [[atmosphere-natural-log-pressure-coordinate]]
 ===  Atmosphere natural log pressure coordinate
 
-
 ----
 standard_name = "atmosphere_ln_pressure_coordinate"
 ----
 
-
-
-Definition:::
+Definition::: 
 +
 ----
-
 p(k) = p0 * exp(-lev(k))
-
 ----
 
 where **`p(k)`** is the pressure at gridpoint **`(k)`**, **`p0`** is a reference pressure, **`lev(k)`** is the dimensionless coordinate at vertical gridpoint **`(k)`**.
 
-The format for the **`formula_terms`** attribute is
-----
+The format for the **`formula_terms`** attribute is 
```

Comment:
Not for this PR, but another style convention we might want to incorporate, e.g. here and in various places elsewhere, and I would advocate for, is to add a colon to the end of statements such as this which flow on into, and refer explicitly to, a code block or similar. As an example which doesn't have to be included here in this PR (in case it isn't clear):

```suggestion
The format for the **`formula_terms`** attribute is:
```